apps/yapms/src/lib/components/modals/importmodal/ImportModal.svelte

```diff
@@ -50,122 +74,100 @@
 						</div>
-					</legend>
-					<input
-						multiple
-						type="file"
-						accept=".geojson, .json"
-						class="file-input file-input-primary w-full"
-						bind:files={geoJsonFiles}
-					/>
-					<button
-						class="btn btn-secondary gap-1 flex-nowrap"
-						on:click={() => {
-							load(importFromGeoJson, geoJsonFiles);
-						}}
-						disabled={!geoJsonFiles || geoJsonFiles.length < 1 || loading}
-					>
-						<ArrowUpTray class="w-5 h-5" />
-						<span>Load</span>
-					</button>
-				</fieldset>
-
-				<div class="divider divider-vertical mt-1 -mb-2"></div>
+					{/if}
 
-				<fieldset class="fieldset w-full flex gap-x-2">
-					<legend class="fieldset-legend flex gap-x-1">
-						<p class="text-lg">Open From Shapefiles</p>
-						<div class="tooltip" data-tip={multipleFilesTooltip}>
-							<DocumentDuplicate class="w-6" />
+					{#if fileTypeIsInvalid}
+						<div class="alert alert-error mt-2">
+							<ExclamationCircle class="w-6 h-6" />
+							<span>Please select a set of GeoJSONs, a set of ShapeFiles or a single SVG.</span>
 						</div>
-					</legend>
-					<input
-						multiple
-						type="file"
-						accept=".shp"
-						class="file-input file-input-primary w-full"
-						bind:files={shapeFiles}
-					/>
-					<button
-						class="btn btn-secondary gap-1 flex-nowrap"
-						on:click={() => {
-							load(importFromShapefiles, shapeFiles);
-						}}
-						disabled={!shapeFiles || shapeFiles.length < 1 || loading}
-					>
-						<ArrowUpTray class="w-5 h-5" />
-						<span>Load</span>
-					</button>
-				</fieldset>
+					{/if}
 
-				<div class="divider divider-vertical mt-1 -mb-2"></div>
-
-				<fieldset class="fieldset w-full flex gap-x-2">
-					<legend class="fieldset-legend">
-						<p class="text-lg">
-							Open from SVG
-							<a
-								class="underline italic text-sm font-light"
-								href="https://github.com/yapms/yapms/wiki/Map-SVG-Format"
-								>Works only with YAPms format</a
-							>
-						</p>
-					</legend>
-					<input
-						type="file"
-						accept=".svg"
-						class="file-input file-input-primary w-full"
-						bind:files={svgFiles}
-					/>
-					<button
-						class="btn btn-secondary gap-1 flex-nowrap"
-						on:click={() => {
-							load(loadSVG, svgFiles);
-						}}
-						disabled={!svgFiles || svgFiles.length < 1 || loading}
-					>
-						<ArrowUpTray class="w-5 h-5" />
-						<span>Load</span>
-					</button>
+					<p class="fieldset-label block">
+						Select multiple geojson or shape files and they will be merged. Only one SVG may be
+						loaded. SVGs must be properly formatted, learn how at
+						<a
+							class="link"
+							href="https://github.com/yapms/yapms/wiki/Map-SVG-Format"
+							target="_blank"
+						>
+							github.com
+						</a>.
+					</p>
 				</fieldset>
 
-				<div class="divider divider-vertical mt-1 mb-1"></div>
+				<ProjectionOptions disabled={loading} />
```

Comment:
I think we should only show this if the files selected are geojson/shapefiles, as setting this options for SVG does nothing.